articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,525 @@
+---
+title: Azure Application Insights for ASP.NET Core applications | Microsoft Docs
+description: Monitor ASP.NET Core web applications for availability, performance, and usage.
+ms.topic: conceptual
+ms.devlang: csharp
+ms.custom: devx-track-csharp
+ms.date: 10/12/2021
+ms.reviewer: casocha
+---
+
+# Application Insights for ASP.NET Core applications
+
+This article describes how to enable Application Insights for an [ASP.NET Core](https://docs.microsoft.com/aspnet/core) application.
+
+Application Insights can collect the following telemetry from your ASP.NET Core application:
+
+> [!div class="checklist"]
+> * Requests
+> * Dependencies
+> * Exceptions
+> * Performance counters
+> * Heartbeats
+> * Logs
+
+We'll use an [ASP.NET Core MVC application](/aspnet/core/tutorials/first-mvc-app) example that targets `net6.0`. You can apply these instructions to all ASP.NET Core applications. If you're using the [Worker Service](/aspnet/core/fundamentals/host/hosted-services#worker-service-template), use the instructions from [here](./worker-service.md).
+
+> [!NOTE]
+> A preview [OpenTelemetry-based .NET offering](./opentelemetry-enable.md?tabs=net) is available. [Learn more](./opentelemetry-overview.md).
+
+[!INCLUDE [azure-monitor-log-analytics-rebrand](../../../includes/azure-monitor-instrumentation-key-deprecation.md)]
+
+## Supported scenarios
+
+The [Application Insights SDK for ASP.NET Core](https://nuget.org/packages/Microsoft.ApplicationInsights.AspNetCore) can monitor your applications no matter where or how they run. If your application is running and has network connectivity to Azure, telemetry can be collected. Application Insights monitoring is supported everywhere .NET Core is supported. Support covers the following scenarios:
+* **Operating system**: Windows, Linux, or Mac
+* **Hosting method**: In process or out of process
+* **Deployment method**: Framework dependent or self-contained
+* **Web server**: IIS (Internet Information Server) or Kestrel
+* **Hosting platform**: The Web Apps feature of Azure App Service, Azure VM, Docker, Azure Kubernetes Service (AKS), and so on
+* **.NET Core version**: All officially [supported .NET Core versions](https://dotnet.microsoft.com/download/dotnet-core) that aren't in preview
+* **IDE**: Visual Studio, Visual Studio Code, or command line
+
+> [!NOTE]
+> ASP.NET Core 3.1 requires [Application Insights 2.8.0](https://www.nuget.org/packages/Microsoft.ApplicationInsights.AspNetCore/2.8.0) or later.
```

Comment:
I would remove this note.
The latest version of the SDK is 2.21 and all older versions are deprecated.

The rest of this article is using .NET 6.
Furthermore, NET Core 3.1 will reach End of Life by the end of this calendar year.

Reply:
Fixed.